pkg/manager/member/scaler.go

```diff
@@ -179,12 +179,23 @@ func ordinalPodName(memberType v1alpha1.MemberType, tcName string, ordinal int32
 // - ordinal: pod ordinal to create or delete
 // - replicas/deleteSlots: desired replicas and deleteSlots by allowing only one pod to be deleted or created
 func scaleOne(actual *apps.StatefulSet, desired *apps.StatefulSet) (scaling int, ordinal int32, replicas int32, deleteSlots sets.Int32) {
+	var ordinals []int32
+	scaling, ordinals, replicas, deleteSlots = scaleMulti(actual, desired, 1)
+	if len(ordinals) == 0 {
+		ordinal = -1
+	} else {
+		ordinal = ordinals[0]
+	}
+	return
+}
+
+func scaleMulti(actual *apps.StatefulSet, desired *apps.StatefulSet, maxCount int) (scaling int, ordinals []int32, replicas int32, deleteSlots sets.Int32) {
```

Comment:
This method only takes effect for "scale in" (not "scale out", should we need to rename it to highlight this?

Reply:
We also need to support scale out multi replicas at the same time.

Reply:
We may have to define different parameters for scale-in and scale-out.